pkgs/tools/security/cariddi/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "cariddi";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "edoardottt";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0cwkycydwndda11m1jszbxchpqabrgspj17y2aj3y3m2x446h27z";
+  };
+
+  vendorSha256 = "0rmiya517i9s4l9nxzwly5vq8cqhhpq66rc7y4sapyaihx20ai3r";
+
+
+  meta = with lib; {
+    description = "Crawler for URLs and endpoints";
+    homepage = "https://github.com/edoardottt/cariddi";
+    license = with licenses; [ gpl3Plus ];
```

Comment:
```suggestion
    license = licenses.gpl3Plus;
```

Reply:
I don't get it what's the issue here? The license is [`GPLv3+`](https://github.com/edoardottt/cariddi/blob/main/output/txt.go#L6-L14).

Reply:
Either is fine. The with should be used when multiple licenses are used. If there is only one license it does not matter.